main.py

```diff
@@ -545,7 +645,22 @@ def print_all_stats(commits_stats: dict, lines_stats: dict, contributors_stats:
     else:
         lines_output = "No SLOC stats, as you've selected at the beginning."
 
-    output = "\n\n".join([contributors_output, '*' * 42, commits_output, '*' * 42, lines_output])
+    if language_repo is not None:
+        language_output = 'Language usage by repository:\n'
+
+        for repo in language_repo:
+            if language_repo[repo]['total'] == 0:
+                continue
+
+            languages = {language:((100 * language_repo[repo][language])/language_repo[repo]['total']) for language in language_repo[repo]}
+            language_output += '\t' f'{repo}: {", ".join(["%s (%.2f%%)" % (k, languages[k]) for k in languages if k != "total"])}\n'
```

Comment:
Try using the limited float precision feature of fstrings
```suggestion
            language_output += f'\t{repo}: {", ".join(["{k} ({languages[k]:.2f}%)" for k in languages if k != "total"])}\n'
```

Reply:
Should be solved in d010048